cmd/oras/internal/option/target.go

```diff
@@ -85,7 +85,7 @@ func (opts *Target) AnnotatedReference() string {
 func (opts *Target) applyFlagsWithPrefix(fs *pflag.FlagSet, prefix, description string) {
 	flagPrefix, notePrefix := applyPrefix(prefix, description)
 	fs.BoolVarP(&opts.IsOCILayout, flagPrefix+"oci-layout", "", false, "set "+notePrefix+"target as an OCI image layout")
-	fs.StringVar(&opts.Path, flagPrefix+"oci-layout-path", "", "set the path for the "+notePrefix+"OCI image layout target")
+	fs.StringVar(&opts.Path, flagPrefix+"oci-layout-path", "", "[Experimental] Set the path for the "+notePrefix+"OCI image layout target")
```

Comment:
It'd be nice if we have some E2E tests for the impacted commands, similar to:

https://github.com/oras-project/oras/blob/8a31d9b19562fe7ca2708e515830348f7e202dff/test/e2e/suite/command/cp.go#L46-L51